stereo-img.js

```diff
@@ -125,6 +125,14 @@ class StereoImg extends HTMLElement {
         } else if(this.type === 'depth') {
           this.stereoData = await parseDepth(this.src);
 
+        } else if(this.type === 'pair') {
+          const righturl = this.getAttribute('rightsrc');
+          this.stereoData = await parseStereoPair(this.src, righturl, {
+            type: this.type,
+            angle: this.angle,
+            projection: this.projection,
+          });
+
         } else {
           // No type specified
```

Comment:
it should be optional to specify `type="pair"`

Can you add logic just below that, if a secondary src attribute is preovided, we use `parseStereoPair`?